server/src/http.rs

```diff
@@ -135,6 +141,61 @@ impl<C: CatalogManager + 'static, Q: QueryExecutor + 'static> Service<C, Q> {
             })
     }
 
+    fn flush_memtable(
+        &self,
+    ) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+        warp::path!("flush_memtable")
+            .and(warp::post())
+            .and(self.with_instance())
+            .and_then(|instance: InstanceRef<C, Q>| async move {
+                let get_all_tables = || {
+                    let mut tables = Vec::new();
+                    for catalog in instance
+                        .catalog_manager
+                        .all_catalogs()
+                        .map_err(|e| Box::new(e) as Box<dyn StdError + Sync + Send>)
```

Comment:
maybe compilier can infer this type
```suggestion
                        .map_err(|e| Box::new(e) as _)
```